locale/areas.zh_CN.tr

```diff
@@ -0,0 +1,135 @@
+# textdomain: areas
+
+
+
+### chatcommands.lua ###
+
+<AreaName>=<保护区名称>
+<ID> [faction_name]=<保护区编号> [派系名称]
+<NewOwner>=<新拥有者>
+<ParentID>=<上级保护区编号>
+<PlayerName>=<玩家名称>
+<newName>=<新名称>
+@1 has given you control over the area "@2" (ID @3).=@1 已将保护区「@2」（编号 @3）的控制权交与您。
+@1 spanning up to @2x@3x@4.=@1，最大为 @2x@3x@4。
```

Comment:
我看了一下源代码，原代码里（`chatcommands.lua`里的`local function size_info`）使用这段文字的方法有点……有问题。我个人认为这里还是先开一个issue。

(EN: The way this string is used in the source code is IMO not particularly suitable for l10n. It would be better to have separate strings for `You can protect areas spanning up to ...` and `Players with the @1 privilege can protect @2 areas spanning up to ...`)

Reply:
看來先擱置這裏的翻譯比較好，等待優化

Reply:
我想了一下，把 "... can protect up to `@2` areas" ... "`@1` spanning up to ..." 翻译成 “…最多可以保护 `@2` 个保护区” … “`@1`，最大为…”问题不大（主要是以后有新的 "... can protect up to `@2` areas" 一类的字符串的话翻译结果会比较怪），所以现在提交 PR，以后再改也可以。